filter/challenges-08.test.js

```diff
@@ -0,0 +1,280 @@
+'use strict';
```

Comment:
Your filter solution is showing up on your reduce branch--be careful, and make sure you check out the master branch before you create the new branch each day.